### PR TITLE
Describe security settings in readme

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by smarterclayton** [comment]
@liggitt review please

File: README.md [hunk 1]
```diff
@@ -35,21 +35,24 @@ NOTE: OpenShift Origin 1.0 has been released [releases page](https://github.com/
...
+OpenShift runs with the following security policy by default:
 
-For more information on the security of containers, see these articles:
-
-* http://opensource.com/business/14/7/docker-security-selinux
-* https://docs.docker.com/articles/security/
-
-Consider using images from trusted parties, building them yourself on OpenShift, or only running containers that run as non-root users.
+* Containers run as a non-root unique user that is separate from other system users
+  * They cannot access host resources, run privileged, or become root
```
Comment (liggitt) [inline README.md]: clarify "host resources"

File: README.md [hunk 2]
```diff
@@ -35,21 +35,24 @@ NOTE: OpenShift Origin 1.0 has been released [releases page](https://github.com/
...
+  * They are given CPU and memory limits defined by the system administrator
+  * Any persistent storage they access will be under a unique SELinux label, which prevents others from seeing their content
+  * These settings are per project, so containers in different projects cannot see each other by default
+* Regular users can run Docker, source, and custom builds
+  * By default, Docker builds can (and often do) run as root. You can control who can create Docker builds through the `builds/docker` and `builds/custom` policy resource.
```
Comment (liggitt) [inline README.md]: @csrwng is there a doc link for configuring build policy?

**Comment by liggitt** [comment]
mostly LGTM, would like a doc link from @csrwng for examples of setting docker/source/custom policy for admins and editors

File: README.md (hunk 2 above)
Comment (csrwng) [inline README.md]: Not yet

**Comment by csrwng** [comment]
I will update the readme with the doc link for build type policy control when that's in.

File: README.md (hunk 1 above)
Comment (smarterclayton) [inline README.md]: Does it need clarification?  I could say "access anything on the host" I huess

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**